build/int.cloudbuild.yaml

```
@@ -104,6 +104,28 @@ steps:
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy budget-local']
 
+# essential contacts
+- id: go-init-essential-contacts-example
+  waitFor:
+    - prepare
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && source_test_env && init_credentials && cd test/integration && RUN_STAGE=init go test -v -run TestEssentialContacsExample ./... -p 1']
```

Now you can use the CLI to simplify this
https://github.com/terraform-google-modules/terraform-google-event-function/pull/87
```suggestion
  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose']
```